### PR TITLE
Completed prep-html-skeleton Noah Khayat

### PR DESCRIPTION
**Pull request by NaeemK1188**
prep-html-skeleton

### CODE REVIEW

**Comment by NaeemK1188** [comment]
Screenshot (357)

File: prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What tags are necessary for a complete HTML Skeleton?
+
+- What type of content belongs within the `<head>` of an HTML document?
+
+- What type of content belongs within the `<body>` of an HTML document?
+
+- Where must the `DOCTYPE` declaration appear in a valid HTML document?
+
```
Comment (shawnkost) [inline prep-html-skeleton/prep-html-skeleton-exercise-quiz-notes.md]: Don't forget to answer the quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
You need to answer the quiz questions as well Noah!

**Comment by NaeemK1188** [comment]
Screenshot (358)

these are the missing quiz answers attached in the image

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**